dev/com.ibm.ws.sipcontainer/src/com/ibm/ws/sip/channel/resolver/dns/impl/chfw/Dns.java

```diff
@@ -82,7 +82,15 @@ public final class Dns {
 	  public static final int NOT_IMPL    = 4;
 	  public static final int REFUSED     = 5;
 	  public static final int TRY_TCP     = 6;
-	  /** 6-15 reserved for future use */
+	  public static final int NOT_ZONE    = 9;
+	  /** more DNS response codes have been defined
+	  *   since this code was written 
+          *   rcode: 7, rcode:8 and rcode: 9
+	  *   we are introducing here rcode: 9  
```

Comment:
I was thinking of linking a number of instead of saying `here`.  I.e  PR `#31163`  or Issue `#31161`

Reply:
Perhaps you can add them in later PRs.

Reply:
I can add issue number in a later PR